packages/accordions/src/styled/accordion/StyledSection.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.section';
+
+export const StyledSection = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
```

Comment:
Is a flex display treatment helpful here (depending on container flex)?

Reply:
I tried a couple of renderings and found that a flex display probably isn't necessary as a default style.